cmd/utils/flags.go

```diff
@@ -1679,6 +1688,15 @@ func SetKlayConfig(ctx *cli.Context, stack *node.Node, cfg *cn.Config) {
 
 	params.OpcodeComputationCostLimit = ctx.GlobalUint64(OpcodeComputationCostLimitFlag.Name)
 
+	if ctx.GlobalIsSet(SnapshotFlag.Name) {
+		cfg.SnapshotCacheSize = ctx.GlobalInt(SnapshotCacheSizeFlag.Name)
+		if cfg.StartBlockNumber != 0 {
+			logger.Crit("State snapshot should not be used with --start-block-num", "num", cfg.StartBlockNumber)
+		}
```

Comment:
How about adding a log informing that snapshot is enabled

Reply:
Updated :) Thanks.